<?xml version="1.0" encoding="UTF-8"?>
<class xmlns="http://xml.phpdox.net/src" full="PhpSpec\Exception\Exception" namespace="PhpSpec\Exception" name="Exception" abstract="false" final="false" start="21" end="43">
  <file path="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/phpspec/phpspec/src/PhpSpec/Exception" file="Exception.php" realpath="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/phpspec/phpspec/src/PhpSpec/Exception/Exception.php" size="856" time="2015-12-17T18:44:10+00:00" unixtime="1450377850" sha1="bca2be2fe3fd47b26f5cc93c574f242483b9cda7" relative="vendor/phpspec/phpspec/src/PhpSpec/Exception/Exception.php"/>
  <docblock>
    <description compact="PhpSpec base exception"/>
  </docblock>
  <extends full="Exception" namespace="" name="Exception"/>
  <member name="cause" visibility="private" line="26">
    <docblock>
      <description/>
      <var type="object">
        <type full="ReflectionFunctionAbstract" namespace="" name="ReflectionFunctionAbstract"/>
      </var>
    </docblock>
  </member>
  <method name="getCause" start="31" end="34" abstract="false" final="false" static="false" visibility="public">
    <docblock>
      <description/>
      <return type="object">
        <type full="ReflectionFunctionAbstract" namespace="" name="ReflectionFunctionAbstract"/>
      </return>
    </docblock>
  </method>
  <method name="setCause" start="39" end="42" abstract="false" final="false" static="false" visibility="public">
    <docblock>
      <description/>
      <param type="object" variable="$cause">
        <type full="ReflectionFunctionAbstract" namespace="" name="ReflectionFunctionAbstract"/>
      </param>
    </docblock>
    <parameter name="cause" byreference="false" type="object">
      <type full="ReflectionFunctionAbstract" namespace="" name="ReflectionFunctionAbstract"/>
    </parameter>
  </method>
</class>
